src/components/AccountContent/index.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+import ProductAvailable from './ProductAvailable'
+import Review from './Review'
+
+const AccountContent = () => {
+  return (
+    <section className="account-content">
+      <ProductAvailable />
```

Comment:
FYI, we have 2 carousels:

Available products
Unavailable products (disable card just showing history)
Can you please update this section?

Ps. I think in design that's only one carousel but in the meeting, we required two.

Reply:
How about `ProductObsolete` for the component name?